### ISSUE TITLE
Hi,I have a question ?

### ISSUE THREAD

**Issue by BCWang93** [abhishekkrthakur/imet-collection#1]
In the utils.py ,what is mean in this line code?
 'self.data = self.data[self.data.fold.isin(folds)].reset_index(drop=True)'
what is mean about 'self.data.fold'?the attribute '.fold' is where?Thanks!

**Comment by abhishekkrthakur** [comment]
i modified the train.csv file and added a new column "fold". you can create this column and make folds 0,1,2,3,4 or you can just modify the code a bit and allow support for file without this column. :)

**Comment by BCWang93** [comment]
> i modified the train.csv file and added a new column "fold". you can create this column and make folds 0,1,2,3,4 or you can just modify the code a bit and allow support for file without this column. :)

Thanks for your reply!I have another question.why this line use '[-5:]'.Thanks!
       ' if len(all_scores[-5:]) == 5:',the lens of all_scores is equal num_epochs?So I can't understand how do you use the last five score?Thanks!Thank you very much!Btw,in this line '    print('Best val loss: {:4f}'.format(best_loss))' in train loop,you seem to forget to update the best_loss value?Thanks!

**Comment by abhishekkrthakur** [comment]
that is my version of early stopping :)
